src/client/hud.h

```diff
@@ -92,6 +92,7 @@ class Hud
 	void drawItem(const ItemStack &item, const core::rect<s32> &rect, bool selected);
 
 	float m_hud_scaling; // cached minetest setting
+	float size_factor;
```

Comment:
```suggestion
	float m_scale_factor_dpi;
```

Reply:
`m_scale_factor` is better. It use `hud_scaling` setting multiplied on DPI.